src/supermarket/config/initializers/paperclip.rb

```diff
@@ -21,7 +21,8 @@
       },
       path: path,
       bucket: ENV['S3_BUCKET'],
-      s3_protocol: ENV['PROTOCOL']
+      s3_protocol: ENV['PROTOCOL'],
+      s3_region: ENV['S3_REGION']
```

Comment:
`s3_region` must be provided for the Paperclip config after the upgrade of the underlying `aws-sdk`. So, an `s3_region` is now required as part of the Supermarket config for storing cookbooks in an S3 bucket.

I suspect we might need to communicate this via a major version bump. As of the writing of this comment on the PR, a Supermarket instance that is currently using an S3 bucket whose administrators do not add an `s3_region` to their settings will _stop_ using the bucket and start storing to local filesystem after an upgrade to a version with this code.

Possibly there could be more checks added around the presence of `S3_BUCKET`, `S3_ACCESS_KEY_ID`, `S3_SECRET_ACCESS_KEY`, and `S3_REGION`. Maybe if any exist, but not all of them, we throw an error at prevent startup?